src/languages/es.ts

```diff
@@ -668,6 +668,7 @@ export default {
             always: 'Inmediatamente',
             daily: 'Cada día',
             mute: 'Nunca',
+            hidden: 'Oculta',
```

Comment:
Can you confirm in Slack whether this should be oculta or oculto

Reply:
[Oculto](https://expensify.slack.com/archives/C01GTK53T8Q/p1706714171806029?thread_ts=1706712766.600179&cid=C01GTK53T8Q)